Features.md

```diff
@@ -13,24 +13,46 @@
    * Profile Photo
    * Email (for authentication related)
    * Hobbies
-   * NUmber (for authentication related)
+   * Number (for authentication related)
    * Kind of places love to visit
    * Current location
-   * User Travel Histroy (the places the user has been to)
+
 * User calendar log : 
-   * Wheter the status of user is free or not on a particular date
-* User tarveling log : 
-   * If going out :
-      * Place going to visit
-      * Is he alone or with a group
-      * Need a group to visit
-      * Invites to be sent (finds friends to sent the Invite)
+   * Current_Status
+   * Current_Date
+   * reminder of journey(gives a reminder on and before jourey date)
+   * color(indicating red for busy, green for free and so on)
+   
+* Planning : 
+  * Date of journey
+  * Duration of journey
+  * Places to visit
+  * Number of mates/friends
+  * Need a group to visit
+  * Invites to be sent (finds friends to sent the Invite)
+
 * Invite table :
    * User-one
    * User-two
    * flag (Accepted or not accepted)
+
 * Matching on the basis of places love to visit
+
+* Diary
+    * Dname
+    * Diary text
+    * Diary Destination
```

Comment:
They are already implemented have a look at it. `diary_brief = models.CharField(max_length=300)
diary_string = models.CharField(max_length=1000,null=True)`